pandas/core/strings.py

```diff
@@ -2977,8 +2977,6 @@ def encode(self, encoding, errors="strict"):
     _shared_docs[
         "str_strip"
     ] = r"""
-    Remove leading and trailing characters.
```

Comment:
It might be simpler to add a substitution here, something like
```suggestion
    Remove %(position)s characters.
```
and then, at the top of the def of `lstrip`, inside `dict`, add `position="leading"` (and similarly "trailing" for `rstrip`)

Reply:
Gotcha, I see. Ok, I'll fix those then commit again. ty!